test/sensitivities/linalg/factorization/cholesky.jl

```diff
@@ -62,7 +21,7 @@
         @test getfield(U, :f) == Base.getproperty
         @test unbox(U) ≈ cholesky(X_).U
 
-        @test_throws ArgumentError ∇(X->cholesky(X).info)(X_)
+        @test_throws Exception ∇(X->cholesky(X).info)(X_)
```

Comment:
Can this be made something more concrete,
maybe a `Union`?
`Exception` will catch `UndefError`

Reply:
```suggestion
        @test_throws MethodError ∇(X->cholesky(X).info)(X_)
```